felt-splash-api/src/main/java/io/github/feltmc/feltapi/api/splash/SplashAPI.java

```diff
@@ -0,0 +1,14 @@
+package io.github.feltmc.feltapi.api.splash;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SplashAPI {
+    public static List<String> splashesToAdd = new ArrayList<>();
+    public static List<String> splashesToRemove = new ArrayList<>();
```

Comment:
Also wouldn't it be better to limit these a bit to prevent them from being overwritten by some daring asshole (me)?

Reply:
oh i am moving to move this to impl, thought you meant something more

Reply:
> oh i am moving to move this to impl, thought you meant something more

Yea, just put the lists in Impl...